### ISSUE TITLE
Add autopkg repo-update all

### ISSUE THREAD

**Issue by gmarnin** [seankaiser/automation-scripts#2]
Thoughts on adding `autopkg repo-update all` to make sure all recipes repos are updated before running autopkg?

**Comment by seankaiser** [comment]
Are you thinking to add this to the script or as part of the setup? If adding it to the script, maybe it should be a CLI argument to update the recipes, but that seems silly since you can just run it from the CLI anyway.

**Comment by gmarnin** [comment]
I was thinking as part of the script. I rarely run the script manually and I only touch the script to update the recipe list. Before the script runs I want to make sure I'm using the current recipe from the github recipe repos.

**Comment by patgmac** [comment]
Updating the repo's could have consequences to the script. For instance, Flip4mac was removed when I updated the repo's today, but I still have it in my autopkg-wrapper list. So I think updating the repo's  should be updated manually so the user can see what has changed and make necessary changes to the script. Unless perhaps the result of repo-update is included in the email as part of the diff.

**Comment by seankaiser** [comment]
Good point. So probably another command line flag to trigger the repo-update would be preferable. Then suggest to users to either manually run with that flag or schedule a new run via another launch daemon to handle that.

**Comment by seankaiser** [comment]
Merged @swy's pull request to implement this (which runs repo-update all each time the script is run.)

 https://github.com/seankaiser/automation-scripts/pull/4
